pandas/core/base.py

```diff
@@ -942,7 +948,22 @@ def argmax(self, axis=None, skipna=True, *args, **kwargs):
 
         See Also
         --------
-        numpy.ndarray.argmax
+        numpy.ndarray.argmax : Returns the indices of the maximum values along an axis.
+
+        Examples
+        --------
+        >>> s = pd.Series(data=[1, None, 5, 4, 5],
+        ...               index=['A', 'B', 'C', 'D', 'E'])
```

Comment:
Do you think you can find some data that looks more "real". In `Series.combine` we've got a small `Series` with animal speeds that I think could be more appropriate to illustrate this: `pd.Series({'falcon': 345.0, 'eagle': 200.0, 'duck': 30.0})`

Also, may be worth adding a note saying that the `2` means that largest value is in the third element, since it's zero-indexed.

Reply:
Sure, I'm gonna use cereal calories dataset as an example and adding that note.